building/build-debs/homeworld-keyserver/src/keycommon/knc/knc.go

```diff
@@ -0,0 +1,53 @@
+package knc
+
+import (
+	"fmt"
+	"os/exec"
+	"encoding/json"
+	"keycommon/reqtarget"
+)
+
+type KncServer struct {
+	Hostname string
+}
```

Comment:
Can you provide a method on Keyserver (AuthenticateWithKerberosTickets) to construct an instance of this?

Reply:
Done